packages/nitro-protocol/test/contracts/NitroAdjudicator/concludePushOutcomeAndTransferAll.test.ts

```diff
@@ -28,6 +28,8 @@ import {
 import {signStates} from '../../../src';
 import {NITRO_MAX_GAS} from '../../../src/transactions';
 
+jest.setTimeout(15_000);
```

Comment:
After a small wild goose chase I have learned that the default timeout for jest tests is 5 seconds.